docs/index.rst

```diff
@@ -67,6 +67,16 @@ Or run the full checks with tox:
     $ tox
 
 
+Upgrading
```

Comment:
I don't think this section is necessary.

Reply:
See comment above on the list of upstream repo release URLs

Reply:
Removed to add to wiki